### DISCUSSION TITLE
Getting undefined symbol crun error

### DISCUSSION THREAD

**Discussion by noor-muradi** [containers/podman#16735] [Q&A]
Hello folks, 
I am getting below curn error on RHEL 9, any suggestion and guidance?

[ahmad@rhel9-2 ~]$ podman run -d -it --name php-app -v /home/ahmad/app/:/app:Z -p 8080:80 docker.io/webdevops/php-apache
Trying to pull docker.io/webdevops/php-apache:latest...
Getting image source signatures
Copying blob 30611fcc1b5e done
Copying blob 5bed26d33875 done
Copying blob f11b29a9c730 done
Copying blob 930bda195c84 done
Copying blob 78bf9a5ad49e done
Copying blob 4c8b85719399 done
Copying blob f70f19977fc5 done
Copying blob 5131985698ec done
Copying blob 65538e0c8933 done
Copying blob d43f884b2c0e done
Copying blob 20d592fa47f1 done
Copying blob d3f32720a7f3 done
Copying blob 2f8bf6213034 done
Copying blob 8356225c2f81 done
Copying config eff3c24c72 done
Writing manifest to image destination
Storing signatures
/usr/bin/crun: symbol lookup error: /usr/bin/crun: undefined symbol: criu_feature_check
ERRO[0158] Removing container f7f815e200f4a97cc48e3f325f447eefdd250cb866edd228e70d5dba310d8775 from runtime after creation failed
Error: OCI runtime error: crun: /usr/bin/crun: symbol lookup error: /usr/bin/crun: undefined symbol: criu_feature_check

`[noor@rhel9-2 ~]$ podman -v
podman version 4.2.0
[noor@rhel9-2 ~]$ buildah -v
buildah version 1.27.2 (image-spec 1.0.2-dev, runtime-spec 1.0.2-dev)
[noor@rhel9-2 ~]$ skopeo -v
skopeo version 1.9.4-dev
[noor@rhel9-2 ~]$ runc -v
runc version 1.1.4
spec: 1.0.2-dev
go: go1.18.4
libseccomp: 2.5.2
[noor@rhel9-2 ~]$ crun -v
crun: symbol lookup error: crun: undefined symbol: criu_feature_check
[noor@rhel9-2 ~]$ whereis crun
crun: /usr/bin/crun /usr/share/man/man1/crun.1.gz
[noor@rhel9-2 ~]$ sudo yum whatprovides crun
[sudo] password for noor:
Updating Subscription Management repositories.
Last metadata expiration check: 0:24:53 ago on Mon 05 Dec 2022 10:29:55 AM IST.
crun-1.4.4-2.el9_0.x86_64 : OCI runtime written in C
Repo        : rhel-9-for-x86_64-appstream-rpms
Matched from:
Provide    : crun = 1.4.4-2.el9_0

crun-1.4.5-2.el9_0.x86_64 : OCI runtime written in C
Repo        : rhel-9-for-x86_64-appstream-rpms
Matched from:
Provide    : crun = 1.4.5-2.el9_0

crun-1.5-1.el9.x86_64 : OCI runtime written in C
Repo        : @System
Matched from:
Provide    : crun = 1.5-1.el9

crun-1.5-1.el9.x86_64 : OCI runtime written in C
Repo        : rhel-9-for-x86_64-appstream-rpms
Matched from:
Provide    : crun = 1.5-1.el9`

**Comment by flouthoc** [comment]
Looks similar to this https://github.com/containers/crun/issues/1085
